bootstrap.sh

```diff
@@ -32,13 +32,16 @@ function install_toolchain {
             rm -r swift-$SWIFT_VERSION-$RELEASE-$SWIFT_TARGET.pkg
             ;;
         ubuntu*)
+            # select toolchain binary based on cpu architecture
+            if [[ $(arch) = aarch* ]]; then ARCH=-$(arch); fi
+
             mkdir -p Toolchains/swift-$SWIFT_VERSION-$RELEASE.xctoolchain
             # download
-            curl -LO https://download.swift.org/swift-$SWIFT_VERSION-$BRANCH/$LINUX_DISTRO/swift-$SWIFT_VERSION-$RELEASE/swift-$SWIFT_VERSION-$RELEASE-$SWIFT_TARGET.tar.gz
+            curl -LO https://download.swift.org/swift-$SWIFT_VERSION-$BRANCH/$LINUX_DISTRO$ARCH/swift-$SWIFT_VERSION-$RELEASE/swift-$SWIFT_VERSION-$RELEASE-$SWIFT_TARGET$ARCH.tar.gz
```

Comment:
Universal binaries are not available on linux, so `-aarch64` is appended when building on arm